packages/@uppy/companion/src/companion.js

```diff
@@ -117,7 +117,7 @@ module.exports.app = (optionsArg = {}) => {
   app.use(middlewares.cors(options))
 
   // add uppy options to the request object so it can be accessed by subsequent handlers.
-  app.use('*', middlewares.getCompanionMiddleware(options))
+  app.use(/(.*)/, middlewares.getCompanionMiddleware(options))
```

Comment:
I thought this should be
```suggestion
  app.use(middlewares.getCompanionMiddleware(options))
```